pages/data-providers/[data-provider-id]/index.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+
+const Index = () => {
+  const router = useRouter()
+
+  useEffect(() => {
+    const pathname = `${router.pathname}/overview`
+    const slash = router.asPath.substr(-1) !== '/' ? '/' : ''
+    const asPath = `${router.asPath}${slash}overview`
+
+    router.replace(pathname, asPath)
+  })
```

Comment:
You should pass empty array as a second parameter to match `componentDidMount` method.

Reply:
To useEffect, you mean?

Reply:
yes